src/common/utxobased/db/Processor.ts

```diff
@@ -69,6 +71,14 @@ export interface Processor {
 
   updateAddressByScriptPubKey(scriptPubKey: string, data: Partial<IAddress>): void
 
+  insertAddressPathBySPubKey(scriptPubKey: string, addressPath: string): Promise<void>
```

Comment:
What is that? 
It looks like there is no implementation for this method.

Reply:
Damn, this slipped in while testing some unrelated code. Removed.